### DISCUSSION TITLE
For production sites, we'd need a real (and working) duplication features

### DISCUSSION THREAD

**Discussion by stouch** [directus/directus#9910] [Feature Requests]
Hello,

First of all, I want to say that I love Directus. BUT : 

"Save as copy" does not still to work for O2M fields (it does not copy anything, it just move related rows to the duplicated item), or M2M fields. Deselecting ID, or ID in O2M sub-fields, in duplication fields, does not solve the problem.

Actually, we cannot use Directus in Production because it is nearly impossible to manage a lot of contents without a real duplication feature.

Is something planned in a next version about this ?

Thank you, Have a great day.

**Comment by benhaynes** [comment]
All issues are being triaged and worked on, but we can't guarantee a timeline for fixes unless they are sponsored. Other than that, giving issues a 👍 or meaningful comment helps us know the biggest pain points.

**Comment by azrikahar** [comment]
This should be resolved by #10418
